### PR TITLE
Hook generator

### PR DESCRIPTION
**Pull request by kristian240**
* Add hook generator plop

### CODE REVIEW

File: plop/hook/README.md.hbs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+npm i @infinum/{{dashCase hookName}}
+```
+
+## Contribution
+
+Yes please! See the [contributing guidelines](https://github.com/infinum/react-nuts-and-bolts/blob/master/CONTRIBUTING.md) for details.
+
+## Licence
+
+This project is licensed under the terms of the [MIT license](https://github.com/infinum/react-nuts-and-bolts/blob/master/LICENSE).
```
Comment (isBatak) [inline plop/hook/README.md.hbs:21]:
```suggestion
This project is licensed under the terms of the [MIT license](https://github.com/infinum/react-nuts-and-bolts/blob/master/LICENSE).

# Credits

`{{camelCase hookName}}` is maintained and sponsored by
[Infinum](https://www.infinum.com).

<p align="center">
  <a href='https://infinum.com'>
    <picture>
        <source srcset="https://assets.infinum.com/brand/logo/static/white.svg" media="(prefers-color-scheme: dark)">
        <img src="https://assets.infinum.com/brand/logo/static/default.svg">
    </picture>
  </a>
</p>
```

File: plop/hook/tests/{{camelCase hookName}}.test.ts.hbs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
+ * Testing Checklist:
+ * - Common use cases snapshotted
+ * - Common use cases run through axe/toHaveNoViolations
+ * - role/aria/data attributes tested
+ * - Component behaviors tested (reacts to events, handles callbacks appropriately, updates state correctly, etc.)
+ * - Controlled/uncontrolled use cases tested
+ * - Associated utils/helpers/etc. tested
+ */
```
Comment (isBatak) [inline plop/hook/tests/{{camelCase hookName}}.test.ts.hbs:9]: Do we need this description for hook?

Comment (kristian240) [reply plop/hook/tests/{{camelCase hookName}}.test.ts.hbs:9]: No, leftover

**Review by isBatak — APPROVED** [review]

File: plop/hook/tests/{{camelCase hookName}}.test.ts.hbs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { renderHook } from "@infinum/test-utils";
```
Comment (isBatak) [inline plop/hook/tests/{{camelCase hookName}}.test.ts.hbs:1]: Hehe we need to create this lib too `@infinum/test-utils`

Comment (isBatak) [reply plop/hook/tests/{{camelCase hookName}}.test.ts.hbs:1]: It's my leftover

Comment (isBatak) [reply plop/hook/tests/{{camelCase hookName}}.test.ts.hbs:1]: also, it should be named `react-test-utils`... or should we have a specific org like `@infinum-react/`

**Pull request merged**